grpc-protobuf/logging.proto

```diff
@@ -11,6 +12,9 @@ service DoguLogMessages {
   // QueryForDogu queries logs for a dogu. This api-call is only available in a MultiNode CES with k8s-ces-control.
   // The logs are queried from k8s-loki and therefore it is only possible to query logs from period of max 30 days.
   rpc QueryForDogu(DoguLogMessageQueryRequest) returns (stream DoguLogMessage);
+
+  // SetLogLevel sets the log level for a specific dogu.
+  rpc SetLogLevel(LogLevelRequest) returns (google.protobuf.Empty);
 }
 
 message DoguLogMessageRequest {
```

Comment:
Like above, I suggest to add doc comments also on top level symbols.

Reply:
I have also added a doc here.